Add `helperText` prop to `Field` and `FieldContainer` (#1562)
vivid-planet/comet: .changeset/eleven-impalas-attack.md

---
"@comet/admin": minor
---

> `Field` now supports a `helperText` prop to provide information

Reviewer: ```suggestion
Add `helperText` prop to `Field` and `FieldContainer` to provide additional information
```

> `Field` now supports a `helperText` prop to provide information.

Reviewer 2: ```suggestion
`Field` now supports a `helperText` prop to provide information
```

We usually don't end the changelog header with a dot
